sopel/plugins/rules.py

```diff
@@ -465,16 +465,25 @@ def __init__(self) -> None:
 
     def start(self) -> None:
         """Record a starting time (before execution)."""
-        self.started_at = datetime.datetime.utcnow()
+        self.started_at = datetime.datetime.utcnow().replace(tzinfo=datetime.timezone.utc)
```

Comment:
Why not `pytz.utc.localize(datetime.datetime.utcnow())`?

Reply:
Mostly that I didn't see any usage of it in the machinery that I poking around in, I patterned the `replace()` on the [code in `Trigger`](https://github.com/SnoopJ/sopel/blob/19605f9509f0471a1688e7cf6fc64f00790af970/sopel/trigger.py#L192-L195) which we may want to change in this PR. I'll defer to your judgment about that.